css/build-css-testsuites.sh

```diff
@@ -1,8 +1,7 @@
 #!/usr/bin/env sh
 set -ex
 
-SCRIPT_DIR=$(dirname $(readlink -f "$0"))
-WPT_ROOT=$(readlink -f $SCRIPT_DIR/..)
+WPT_ROOT=$(cd $(dirname "$0")/.. && pwd -P)
```

Comment:
In general, this is wrong, though I expect we can assume the `css` directory isn't a symlink.

Really what we want is `SCRIPT_DIR=$(cd $(dirname "$0") && pwd -P)` and then keep WPT_ROOT as is.

Reply:
Can you give an example of when the result would be different, confirned on command line?

Reply:
Note that leaving `WPT_ROOT` unchanged would mean still using `readlink -f`, which would make this change pointless :)

Reply:
Alright, so I tested and if `x` is a symlink to `y`, then `cd x/..` will end up in the parent directory of the symlink `x` rather than `y`'s parent directory. If `css` for some odd reason were a symlink in this repo that would actually be the right thing, but if someone has a symlink outside of the checkout pointing to the `css` directory they can't invoke the script via that symlink. So, uh, I guess I'll first establish a symlink-free `SCRIPT_DIR` and then use that...

Reply:
I think having `WPT_ROOT=$SCRIPT_DIR/..` should be fine?

Reply:
That's what I went with